e2e/openOsmWithUser.spec.ts

```diff
@@ -8,27 +8,42 @@ let projects = getProjects();
 projects = projects.filter((arg) => arg.name !== 'atlas');
 
 for (const project of projects) {
-  test(`As User with no rights, I can go to ${project.title}, open map and open Rapid OSM editor at map coordinates`, async ({
-    context,
-    pageManager,
-  }) => {
-    await pageManager.atBrowser.openProject(project, { skipCookieBanner: true });
-    await pageManager.fromNavigationMenu.goToMap();
-
-    // TO DO: remove this action after 18582 issue is fixed
-    await pageManager.atMap.goToSpecificAreaByUrl(10.597, 53.9196, 27.5097, project);
-
-    const coordinates = await pageManager.atMap.getViewportFromUrl();
-
-    const [newPage] = await Promise.all([
-      context.waitForEvent('page'),
-      (await pageManager.atToolBar.getButtonByText('Edit map in OSM')).click({
-        delay: 330,
-      }),
-    ]);
-
-    await pageManager.atMap.waitForUrlToMatchPattern(/rapideditor/, newPage);
-    const osmCoordinates = await pageManager.atMap.getViewportFromUrl(newPage);
-    expect(osmCoordinates).toStrictEqual(coordinates);
-  });
+  const editors: [string, RegExp][] = [
+    ['OpenStreetMap.org default editor', /openstreetmap/],
```

Comment:
we should also check a case when there is no editor set. For that case, we need to edit response of the API (we had discussion here https://konturio.slack.com/archives/C037ZB1Q53P/p1723459994598479?thread_ts=1723299542.327509&cid=C037ZB1Q53P, so take a look how to do it)

Reply:
this situation happens when user has just been created. If now by default after registration we set user OSM (so he/she does not have an empty field in the dropdown), then skip this comment

Reply:
I'm going to add a test for this case